src/App.js

```diff
@@ -1,6 +1,10 @@
+/* eslint-disable import/extensions */
 import { Component } from 'react';
+import { Routes, Route } from 'react-router-dom';
 import './App.css';
-import Calculator from './components/calculator';
+import Calculator from './components/Calculator';
```

Comment:
- [ ] When running your app I got an error, please kindly fix it 🙃 😉 

<img width="636" alt="Screen Shot 2022-09-13 at 14 32 12" src="https://user-images.githubusercontent.com/74501523/189902653-a19d8008-9d32-4242-a094-093d7fc63378.png">